src/renderer/vt/state.cpp

```diff
@@ -509,3 +509,17 @@ void VtEngine::SetResizeQuirk(const bool resizeQuirk)
 {
     return S_OK;
 }
+
+// Method Description:
+// - Manually emit a "Erase All" sequence to the connected terminal. When the
+//   client requests a Erase All operation, we need to manually tell the
+//   connected terminal to do the same thing, so that the terminal will move
+//   it's own buffer contents into the scrollback.
+// Arguments:
+// - <none>
+// Return Value:
+// - S_OK if we wrote the sequences successfully, otherwise an appropriate HRESULT
+[[nodiscard]] HRESULT VtEngine::ManuallyClearViewport() noexcept
+{
+    return _ClearScreen();

```

Comment:
Should we just skip the middle-man and make `ClearScreen` public?

Reply:
Eh, I just got rid of this entirely